config/linux-settings.gypi

```diff
@@ -71,7 +71,7 @@
 					'-Wno-unused-parameter',	# Just contributes build noise
 					'-Werror=return-type',
 					'-Werror=uninitialized',
-					'-Wno-error=maybe-uninitialized',
+					#'-Wno-error=maybe-uninitialized',
```

Comment:
I suggest removing the comment line if we are requiring errors for this case now. Do we need to double check builds on all platforms here, or is this a GCC specific warning/error?